lib/hooks/index.js

```diff
@@ -97,25 +58,44 @@ function findModuleVersion(request, parent, load) {
     if (mainScriptDir.indexOf(paths[i]) === 0) {
       // This is unsafe on windows if request contains a namespace
       // TODO: convert path separators in request
-      var pjson = load(path.join(paths[i], request, 'package.json'));
-      return pjson.version;
+      return path.join(paths[i], request);
     }
   }
   return null;
 }
 
+function findModuleVersion(modulePath, load) {
```

Comment:
This modulePath cannot be empty. This is the absolute path to the module. I have added documentation to this effect.